packages/core/admin/admin/src/translations/de.json

```diff
@@ -450,13 +635,45 @@
   "content-manager.popUpWarning.warning.unpublish-question": "Willst du sicher die Veröffentlichung dieses Inhalts rückgängig machen?",
   "content-manager.popUpWarning.warning.updateAllSettings": "Dadurch werden alle deine Einstellungen geändert.",
   "content-manager.popUpwarning.warning.has-draft-relations.button-confirm": "Ja, veröffentlichen",
+  "content-manager.popUpwarning.warning.has-draft-relations.message": "<b>{count, plural, =0 { of your content relations is} one { of your content relations is} other { of your content relations are}}</b> sind noch nicht veröffentlicht.<br></br>Das kann zu kaputten Links und Fehlern in deinem Projekt führen.",
+  "content-manager.popover.display-relations.label": "Beziehungen darstellen",
   "content-manager.success.record.delete": "Gelöscht",
   "content-manager.success.record.publish": "Veröffentlicht",
   "content-manager.success.record.save": "Gespeichert",
   "content-manager.success.record.unpublish": "Veröffentlichung zurückgenommen",
+  "content-manager.utils.data-loaded": "The {number, plural, =1 {entry has} other {entries have}} wurden erfolgreich geladen",
```

Comment:
```suggestion
  "content-manager.utils.data-loaded": "Der {number, plural, =1 {Eintrag wurde} other {Einträge wurden}} erfolgreich geladen",
```

Reply:
Shouldn't that change to "**Die** 5 Einträge wurden erfolgreich geladen" for plural strings?

Reply:
I'm sorry, you are right :)